lib/dialects/abstract/query-generator.js

```diff
@@ -1413,7 +1428,13 @@ class QueryGenerator {
         } else {
           deprecations.noRawAttributes();
         }
-        attr = [attr[0], this.quoteIdentifier(attr[1])].join(' AS ');
+        let alias = attr[1];
+
+        if (this.options.minifyAliases) {
+          alias = this._getMinifiedAlias(alias, mainTableAs, options);
+        }
+
+        attr = [attr[0], this.quoteIdentifier(alias)].join(' AS ');
```

Comment:
Why is it necessary to minify something inside a `escapeAttributes` function?

Reply:
This function is only used for the attributes of the main query ([here](https://github.com/sequelize/sequelize/pull/11095/files#diff-def52474024b12c7d133bb504cdf57d1R1168)).
I don't really know why this is a different function than the one used for includes or subQueries.